torii/src/routing.rs

```diff
@@ -179,7 +180,7 @@ async fn handle_post_configuration(
 }
 
 #[iroha_futures::telemetry_future]
-async fn handle_blocks_stream(kura: Arc<Kura>, mut stream: WebSocket) -> eyre::Result<()> {
+pub async fn handle_blocks_stream(kura: Arc<Kura>, mut stream: WebSocket) -> eyre::Result<()> {
```

Comment:
Why this functions have to be public now?

Reply:
Because `Torii::new` now defined next to `Torii` itself, in the parent module.